### PR TITLE
Try To Update Existing SChinese and TChinese Translation,Fix A Little Problem Of Korean Translation

### PR DESCRIPTION
**Pull request by Zout141**
Update Existing SChinese Translation,
Fully Update Existing tChinese Translation,
And also fix A Little Problem Of Korean Translation

### CODE REVIEW

File: launcher/game/tl/schinese/launcher.rpy [hunk 1]
```diff
@@ -1853,3 +1853,159 @@
...
+    old "Ignore until next launch."
+    new "忽略直到下次启动"
+
+    # game/editor.rpy:607
+    old "Do not ask again."
+    new "别再问了"
+
+    # game/front_page.rpy:133
+    old "The Question"
+    new "问题"
```
Comment (Dobby233Liu) [inline launcher/game/tl/schinese/launcher.rpy:1927]: Left untranslated in `the_question/game/tl/schinese/options.rpy`

File: launcher/game/tl/schinese/common.rpy [hunk 2]
```diff
@@ -803,3 +803,339 @@
...
+    old "mute all"
+    new "屏蔽全部"
+
+    # renpy/common/00speechbubble.rpy:397
+    old "(hide)"
+    new "(hide)"
+
+    # renpy/common/00speechbubble.rpy:408
+    old "(clear retained bubbles)"
+    new "(clear retained bubbles)"
```
Comment (CloneWith) [inline launcher/game/tl/schinese/common.rpy:1105]: These strings are translatable.

```suggestion
    # renpy/common/00speechbubble.rpy:397
    old "(hide)"
    new "(隐藏)"

    # renpy/common/00speechbubble.rpy:408
    old "(clear retained bubbles)"
    new "(清除保留的气泡)"
```

File: launcher/game/tl/schinese/developer.rpy [hunk 3]
```diff
@@ -96,3 +96,42 @@
...
+    old "Hide Filename and Line"
+    new "隐藏文件名和行"
+
+    # renpy/common/_developer/developer.rpym:127
+    old "Layer [l]:"
+    new "层  [l] ："
+
+    # renpy/common/_developer/developer.rpym:131
+    old "    [name] [attributes] (hidden)"
+    new "    [name] [attributes] (hidden)"
```
Comment (CloneWith) [inline launcher/game/tl/schinese/developer.rpy:121]:
```suggestion
    # renpy/common/_developer/developer.rpym:131
    old "    [name] [attributes] (hidden)"
    new "    [name] [attributes] (隐藏)"
```

File: launcher/game/tl/schinese/launcher.rpy [hunk 4]
```diff
@@ -1853,3 +1853,159 @@
...
+    old "Prefer the web documentation"
+    new "偏好网络文档"
+
+    # game/preferences.rpy:262
+    old "Restore window position"
+    new "恢复窗口位置"
+
+    # game/preferences.rpy:266
+    old "Prefer RPU updates"
+    new "首选 RPU 更新"
```
Comment (CloneWith) [inline launcher/game/tl/schinese/launcher.rpy:1967]:
```suggestion
    # game/preferences.rpy:266
    old "Prefer RPU updates"
    new "偏好 RPU 更新"
```
To use the existing translation above.

**Comment by renpytom** [comment]
@zedraxlo Any chance you can review these changes?

**Comment by zedraxlo** [comment]
> @zedraxlo Any chance you can review these changes?

@renpytom I have replied to you via E-mail.

**Comment by renpytom** [comment]
@Zout141 Is machine translation being used here?

**Comment by renpytom** [comment]
Is this ready for merge?

**Comment by renpytom** [comment]
Merged, thank you.

**Pull request merged**